tests/ui/delegation/empty-glob.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(fn_delegation)]
+#![allow(incomplete_features)]
+
+trait Trait {}
+
+struct S;
+impl S {
+    reuse Trait::*; //~ ERROR empty glob delegation is not supported
```

Comment:
In this case it's annoying to support because `Trait` (and possible generic arguments in it), still needs to be checked for stability, privacy, etc.
So we'll have to keep an otherwise unnecessary dummy item after expansion, and even in HIR, to perform these checks.